lib/libutee/include/tee_api_defines.h

```diff
@@ -449,9 +449,14 @@
  *      return TEE_ERROR_BAD_PARAMETERS;
  *  }
  */
-#define TEE_PARAM_TYPES(t0,t1,t2,t3) \
-   ((t0) | ((t1) << 4) | ((t2) << 8) | ((t3) << 12))
-
+#ifdef CFG_TEE_BENCHMARK
```

Comment:
i think you do not need to patch this routine.

since the "benchmark" support is handled from libutee, i think the libutee should get the bench buffer arg and remove it from  the "param_types" argument before branching to TA "Open" and "Invoke" entry points.
This way, Tursted Application is not bothered with this addition parameter, and TA must use the "BENCH" macros for adding any timestamps, without wondering about the extra "param".